background.js

```diff
@@ -144,4 +144,66 @@ function setBadge(title, text, badgeColor, alienIcon, tab) {
     })
 }
 
+function getSnoowrap(interactive, callback) {
+    'use strict';
+    var clientId = 'JM8JSElud0Rm1g';
```

Comment:
This value has to be replaced with your own client ID. To get this value, go in to the preferences section of your Reddit account and click [apps](https://www.reddit.com/prefs/apps/). Create an installed app and set its redirect url to `https://<CHROME_EXTENSION_ID>.chromiumapp.org/provider_cb`. After creating the app you should see a client ID which you can copy-paste into this line